ecllibrary/std/Crypto.ecl

```diff
@@ -143,15 +169,70 @@ EXPORT PublicKeyEncryption(VARSTRING pkAlgorithm, VARSTRING publicKeyFile = '',
     END;
 END; // PublicKeyEncryption module
 
+/**
+ * Encryption module containing asymmetric encryption/decryption/digital
+ * signing/signature verification functions
+ *
+ * @param   pkAlgorithm    ASymmetric algorithm to use, as returned by SupportedPublicKeyAlgorithms()
+ * @param   publicKeyFile  File specification of PEM formatted public key file
+ * @param   privateKeyFile File specification of PEM formatted private key file
+ * @param   passphrase     Passphrase buffer to use for encryption/encryption/signing/verifying
```

Comment:
minor: "Binary passphrase" instead of "Passphrase buffer"?

Reply:
It doesn't HAVE to be binary, so reworded from "Passphrase buffer" to "Passphrase"